src/spell.c

```diff
@@ -160,72 +158,105 @@ char ilet;
 
 /* TRUE: book should be destroyed by caller */
 STATIC_OVL boolean
-cursed_book(bp)
-	struct obj *bp;
-{
+cursed_book(struct obj *bp){
```

Comment:
The standard for parameters is to put only their name here, and put their types below:
```
boolean
cursed_book(bp)
struct obj * bp;
{
```

Reply:
K&R style func decs are not used by anybody except for consistency with existing code. I'm not particularly concerned about that, and I don't really feel like keeping it k&r forever is a good idea anyway ([see here, which is my token anti-K&R link](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=48091)).